locales/en-US/strings.properties

```diff
@@ -115,6 +115,19 @@ time_label_day={number}d
 # in English, while "Home" should be localized matching the about:preferences
 # sidebar mozilla-central string for the panel that has preferences related to
 # what is shown for the homepage, new windows, and new tabs.
+prefs_new_windows_tabs_header=New Windows and Tabs
+prefs_new_windows_description=Choose what you see when you open your homepage, new windows, and new tabs.
+
+prefs_homepage_choice_label=Homepage and new windows:
+prefs_newtab_choice_label=New tabs:
+prefs_homepage_newtab_choice_1_firefox_home=Firefox Home
+prefs_homepage_newtab_choice_2_custom=Custom URL(s)
+prefs_homepage_newtab_choice_3_blank=Blank Page
+
+prefs_homepage_custom_url_placeholder=Paste a URL
+prefs_homepage_custom_url_current_pages_button=Use Current Pages
```

Comment:
Do we need a plural here? Firefox settings have "Use Current Page" and "Use Current Pages", depending on the number of tabs.

Reply:
Good point, I see code for that in the existing UI actually, I will update this